test/integration/tasks/build-extensions.js

```diff
@@ -13,5 +13,5 @@ module.exports = async ({ testLocalSdk }) => {
     }
   }
 
-  await Promise.all(['test-field-extension', 'test-page-extension'].map(buildExtension))
+  await Promise.all(['test-extension'].map(buildExtension))
```

Comment:
I installed a page-extension extension to the test space, I can clean it up if you haven't already.

Reply:
No worries, I'll do a cleanup